app/profiles/[landlordId]/page.tsx

```diff
@@ -53,7 +53,7 @@ export default async function landlordProfilePage( {params }: { params: { landlo
         const { data: details, error: propertyDetailsError } = await supabase
         .from('properties')
         .select('address')
-        .in('id', landlordProperties.map(property => property.property_id))
+        .in('id', landlordProperties.map((property: { property_id: string }) => property.property_id))
```

Comment:
this is not needed as the database types already specify that properties have a property_id.

I'm guessing that you added this as TS complained about this before you had run the type generation. But now that the type file is up to date, this addition does nothing, and makes it harder to catch potential issues that could arise from future changes to the database as TS will miss them at build time.